libkbfs/md_journal.go

```diff
@@ -464,7 +464,10 @@ func (j *mdJournal) convertToBranch(
 		brmd.SetUnmerged()
 		brmd.SetBranchID(bid)
 
-		// Delete the old "merged" version from the cache.
+		// Delete the old "merged" version from the cache.  We aren't
+		// equipped here to cache the new version, but it will
+		// eventually get passed via an mdFlushListener to somewhere
+		// that will cache it.
```

Comment:
Maybe your thought is that, if it's not already in the cache, we'd have to fetch it from the server anyway once the journal is flushed and CR is done?  That's a fair point.